po/pt_BR.po

```diff
@@ -132,12 +132,12 @@ msgstr "Nenhum Alarme"
 
 #: src/Widgets/AlarmTimeWidget.vala:74
 msgid "Click the add icon in the toolbar below to get started."
-msgstr ""
+msgstr "Clique no ícone de adicionar na barra inferior para começar."
 
 #. action buttons
 #: src/Widgets/AlarmTimeWidget.vala:93
 msgid "Add…"
-msgstr ""
+msgstr "Adicionar"
```

Comment:
```suggestion
msgstr "Adicionar…"
```

Same here. In general, "…" implicates that the app will ask you input in the next screen/view/dialog. In this context, the app will show a dialog that users can set a title/date/time of an alarm.